INSTALLATION.md

```diff
@@ -109,6 +110,31 @@ Install the packages required by `talawa-api` using this command:
 npm install
 ```
 
+## Installation using Docker
```

Comment:
1. This needs to be a completely separate section. The existing section needs to be renamed to clearly show that it's not using Docker, the Docker section should then follow afterwards.
2. The table of contents must show this differentiation too with the proper indentation.


We need to make the documentation as accessible as possible.

Reply:
Moved Docker Installation Steps to separate section and updated the table of contents